src/App.tsx

```diff
@@ -10,33 +11,22 @@ export const tabs = [
 ];
 
 export const App: React.FC = () => {
-  return (
-    <div className="section">
-      <h1 className="title">
-        Selected tab is Tab 1
-      </h1>
-
-      <div data-cy="TabsComponent">
-        <div className="tabs is-boxed">
-          <ul>
-            <li className="is-active" data-cy="Tab">
-              <a href="#tab-1" data-cy="TabLink">Tab 1</a>
-            </li>
+  const [selectedTabId, setSelectedTabId] = useState(1);
 
-            <li data-cy="Tab">
-              <a href="#tab-2" data-cy="TabLink">Tab 2</a>
-            </li>
+  const setTab = (tabId: string) => {
+    const id = tabId;
 
-            <li data-cy="Tab">
-              <a href="#tab-3" data-cy="TabLink">Tab 3</a>
-            </li>
-          </ul>
-        </div>
+    setSelectedTabId(+id);
+  };
 
-        <div className="block" data-cy="TabContent">
-          Some text 1
-        </div>
+  return (
+    <>
+      <div className="section">
+        <h1 className="title">
+          {`Selected tab is Tab ${selectedTabId}`}
+        </h1>
       </div>
-    </div>
+      <Tabs tabs={tabs} handler={setTab} selectTab={selectedTabId} />
```

Comment:
When you have 3 or more arguments - better write each from new line.
```suggestion
      <Tabs 
        tabs={tabs} 
        handler={setTab} 
        selectTab={selectedTabId} 
      />
```